spec/lib/tasks/bulk_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "bulk" do
+  describe "email" do
+    before do
+      Rake::Task["bulk:email"].reenable
+      allow(ENV).to receive(:fetch).with("SUBJECT").and_return("subject")
```

Comment:
This repo [has ClimateControl](https://github.com/alphagov/email-alert-api/blob/master/Gemfile#L29), so we should use that here.

Reply:
[Switched to ClimateControl](https://github.com/alphagov/email-alert-api/commit/2ee6d188f09d57f52f547629c8f5f9864a518752)